server/jetstream_cluster.go

```diff
@@ -8379,7 +8393,10 @@ RETRY:
 	releaseSyncOutSem()
 
 	if n.GroupLeader() == _EMPTY_ {
-		return fmt.Errorf("catchup for stream '%s > %s' aborted, no leader", mset.account(), mset.name())
+		return &catchupErr{
```

Comment:
Could you wrap withut creating a custom type using:

```
return fmt.Errorf("....... : %w", ..., errCatchupAbortedNoLeader)
```

Reply:
Done